airflow/logging_config.py

```diff
@@ -54,8 +58,8 @@ def configure_logging():
     else:
         logging_class_path = 'airflow.config_templates.' \
                              'airflow_local_settings.DEFAULT_LOGGING_CONFIG'
```

Comment:
```suggestion
        logging_class_path = 'airflow.logging_config.DEFAULT_LOGGING_CONFIG'
```

Reply:
I think we can just remove this line entirely since the CONFIG is now in this file itself

```suggestion
```

Reply:
I think this is still needed for the "reload logging config" hack we pull in a few places. (Dag file processor is the one that comes to mind)